src/components/ImageUploadComponent.tsx

```diff
@@ -68,11 +68,9 @@ export class ImageUploadComponent extends React.Component<InlineComponentProps,
                     {this.state.inProgress &&
                         <img src="./images/loadingGif.gif" style={{ width: "112px", textAlign: "center" }} />}
                     {this.props.currentModel &&
-                        <>
```

Comment:
The code actually has empty tag ? How ?

Reply:
I added unknowingly, after that realised that it is not required.